lib/ssl/src/dtls_gen_connection.erl

```diff
@@ -577,10 +580,15 @@ unprocessed_events(Events) ->
     erlang:length(Events)-1.
 
 encode_handshake_flight(Flight, Version, MaxFragmentSize, Epoch, ConnectionStates) ->
-    Fragments = lists:map(fun(Handshake) ->
-				  dtls_handshake:fragment_handshake(Handshake, MaxFragmentSize)
-			  end, Flight),
-    dtls_record:encode_handshake(Fragments, Version, Epoch, ConnectionStates).
+    Encode = fun(Fragment, {Acc, Cs0}) ->
+                     {Enc, Cs} = dtls_record:encode_handshake(Fragment, Version, Epoch, Cs0),
+                     {[Enc|Acc], Cs}
+             end,
+    {Rev, Cs} = lists:foldl(fun(Handshake, {Acc,Cs0}) ->
+                                    Fs = dtls_handshake:fragment_handshake(Handshake, MaxFragmentSize),
```

Comment:
Could we use Frags instead of Fs if you think Fragments is too long?  Even though the function name fragment_handshake is very helpful here I had to think before I realized what Fs stood for. In this particular case when having MaxFragmentSize also nearby the brain may wrongly associate the s with size before realizing that is not logical.